frontend/src/pages/client-details/SummaryView.vue

```
@@ -40,9 +40,23 @@ const formData = ref<ClientDetails>();
 const isEditing = ref(false);
 const hasAnyChange = ref(false);
 
+const formatBirthdate = (birthdate: string) => new Date(birthdate).toISOString().split("T")[0];
+
+const formatClientData = (data: ClientDetails) => {
+  const formattedData = JSON.parse(JSON.stringify(data));
+  if (formattedData.client.birthdate) {
+    try {
+      formattedData.client.birthdate = formatBirthdate(formattedData.client.birthdate);
+    } catch {
+      // ignore - just keeps the original value
```

One small suggestion: I noticed that the catch block in the formatClientData function is empty. While it's fine to continue using the original birthdate when formatting fails, silently swallowing the error could make debugging harder in the future.

Would you consider at least logging a warning or adding a brief comment to clarify the intention? For example:
`catch (error) {
  console.warn("Failed to format birthdate:", formattedData.client.birthdate, error);
}
`
Or, if logging isn't wanted, then add something like this:
`catch {
  // Intentionally ignoring formatting errors – fallback to original value
}
`